### PR TITLE
[WIP] [Feature] Make footer smaller on project related pages

### PR DESCRIPTION
**Pull request by caneruguz**
## Purpose
There was some discussion here about what should happen to the footer  inside project pages because there is less need of a full footer in these cases. 
Discussion (scroll below a few posts: https://github.com/CenterForOpenScience/osf.io/issues/2561 )

This PR is experimenting with a smaller footer in project pages. 

## Changes
- The structure with which we include footer in the mako files is changes and worth a review
- Project pages don't show the main footer links and show the copyright instead

## Side effects
- Because of the change of structure with the footer we need to make sure all pages that needed a footer actually get one. Needs general loading of all osf pages as test or a different method to deal with the footer. 

5bc639b4-e9bb-11e4-9c48-3d2e675ac0f7

### CODE REVIEW

**Comment by GaryKriebel** [comment]
I quite like it.

While it also happens in production, is there any concern having the footer so far up if the page contains very little content?

screen shot 2015-05-07 at 3 42 28 pm

**Comment by caneruguz** [comment]
That's one point of discussion. I don't think a bottom fixed version would look good either in that case because it's so far down. We can try to make sure the content is always properly padded.

**Comment by caneruguz** [comment]
Is there still interest in doing this? @GaryKriebel @JeffSpies @lbanner

File: website/templates/footer.mako [hunk 1]
```diff
@@ -1,8 +1,10 @@
+<%page args="placement"/>
+%if placement != 'projectPage':
 <footer class="footer">
     <div class="container">
         <div class="row">
             <div class="col-sm-2 col-md-2 col-md-offset-1">
-                <h4>OSF</h4>
+                <h4>OSF ${placement}</h4>
```
Comment (sloria) [inline website/templates/footer.mako]: Why should we display placement here?

**Comment by caneruguz** [comment]
@Steve this is not review ready, a lot of code will need to change here

**Comment by caneruguz** [comment]
closing this PR since it's outdated and may not be needed. I'll send another one if we go forward with this.

**Pull request closed without merging**